Jenkinsfile

```diff
@@ -33,13 +33,6 @@ pipeline {
         disableConcurrentBuilds()
         timestamps()
     }
-    triggers {
-        /*
-          Restrict nightly builds to master branch, all others will be built on change only.
-          Note: The BRANCH_NAME will only work with a multi-branch job using the github-branch-source
-        */
-        cron(env.BRANCH_NAME == "master" ? "H H(21-23) * * *" : "")
-    }
```

Comment:
✏️ Delete the PR comment stuff in the setup block

Reply:
I reviewed and the setup block and I don't see anything in their about PRs, so we should be good now.